TestCases/compliance-level-3/0062-feel-mode-function/0062-feel-mode-function-test-01.xml

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!-- Contributed to DMN TCK by StrayAlien -->
+<testCases xmlns="http://www.omg.org/spec/DMN/20160719/testcase" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+    <modelName>0062-feel-mode-function.dmn</modelName>
+    <labels>
+        <label>Data Type: List</label>
+        <label>FEEL Functions: list</label>
+    </labels>
+
+    <testCase id="001">
+        <description>will return median for even list size</description>
+        <resultNode name="decision001" type="decision">
+            <expected>
+                <list>
+                    <item>
+                        <value xsi:type="xsd:decimal">6</value>
+                    </item>
+                </list>
+            </expected>
+        </resultNode>
+    </testCase>
+
+    <testCase id="002">
+        <description>will return median for odd list size</description>
+        <resultNode name="decision002" type="decision">
+            <expected>
+                <list>
+                    <item>
+                        <value xsi:type="xsd:decimal">6</value>
+                    </item>
+                    <item>
+                        <value xsi:type="xsd:decimal">1</value>
```

Comment:
Spec says:

> Returns the mode of the list of numbers. If the result contains multiple elements, they are returned in ascending order.

Hence here it should be from 
```
            <expected>
                <list>
                    <item>
                        <value xsi:type="xsd:decimal">6</value>
                    </item>
                    <item>
                        <value xsi:type="xsd:decimal">1</value>
```

to
```
            <expected>
                <list>
                    <item>
                        <value xsi:type="xsd:decimal">1</value>
                    </item>
                    <item>
                        <value xsi:type="xsd:decimal">6</value>
```

Reply:
Good catch @tarilabs.  Thank you.  I'll bolster the test to have 3 result items to lessen the change of getting it right by 'accident'.

Reply:
@tarilabs amended.